oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 partner-contact https://github.com/OCA/partner-contact 8.0
+intrastat https://github.com/sunflowerit/intrastat 8.0-intrastat_base-tests
```

Comment:
Module `intrastat_base` is available in OCA for V8, so I think you can set the OCA repository `intrastat` here, instead of the sunflowerit branch. Or is it a temporary solution for some reason?